### PR TITLE
Update Test Drive docs with Identity

### PR DESCRIPTION
**Pull request by verythorough**
Addresses part of #600

**- Summary**
Updates the Test Drive docs to use the updated Kaldi template with Netlify Identity and Git Gateway integrated.

**- Test plan**
You can check the GitHub rendering at 
https://github.com/netlify/netlify-cms/blob/test-drive-identity/docs/test-drive.md
(Note that the links to Intro and Quick Start at the end of the doc will not work in this view, due to issues discussed in #511)

I've also pushed a temporary branch to netlify-cms-www to test rendering on netlifycms.org:
https://test-drive-test--netlify-cms-www.netlify.com/docs/test-drive/

**- Description for the changelog**

Update the Test Drive doc to use the newest Kaldi template and Netlify Identity instructions.

**- A picture of a cute animal (not mandatory but encouraged)**
tumblr_n5w3t9msxv1tcrdtfo3_r1_1280

### CODE REVIEW

**Comment by verythorough** [comment]
Note that PR [#52](https://github.com/netlify/netlify-cms-www/pull/52) in netify-cms-www will need to be merged before this PR to ensure proper image display on netlifycms.org.

File: docs/test-drive.md [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
+Netlify CMS can run in any frontend web environment, but the quickest way to try it out is by running it on a pre-configured starter site with Netlify. Our example here is the [Kaldi coffee company template](https://github.com/netlify-templates/one-click-hugo-cms). Use the button below to build and deploy your own copy of the repository:
 
-[![Deploy to Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/netlify-templates/kaldi-hugo-cms-template)
+[![Deploy to Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/netlify-templates/one-click-hugo-cms&stack=cms)
 
-## Authenticate with GitHub
+After clicking that button, you’ll authenticate with GitHub and choose a repository name, then Netlify will automatically clone the template and deploy it, bringing you to the site dashboard when the build is complete. Next, you’ll need to set up Netlify's [Identity](https://www.netlify.com/docs/identity) service to authorize users to log in to the CMS.
```
Comment (eamonnbell) [inline docs/test-drive.md]: Could be it be clearer that Netlify is creating a new repository (with mirrored content) belonging to the authenticated user?

**Comment by eamonnbell** [comment]
I followed the test drive guide to the letter and works fine. See one small comment about use of "clone" in https://github.com/netlify/netlify-cms/pull/626/commits/a890e0da298ec4e6de3daaf93e2bf14e76474044. Looks good!

**Comment by erquhart** [comment]
@verythorough great work on this! In the future we need to delineate more clearly between backend/auth options, as it isn't clear between this section and quick start that two different backends are being used, and what the differences are.

Besides @eamonnbell's comment, this looks good to me.

**Comment by verythorough** [comment]
@erquhart Yeah, kinda fits with the Identity/GG update plan (#600), as well as the overall docs re-org (#598).

@eamonnbell Thanks for checking! I struggled with the description in that paragraph. I wanted to let the user know what's happening, but also didn't want to get too bogged down in the details on something that is supposed to be very simple and require very little of the user. Using "clone" was a cheat to save words, but it could cause more confusion than clarity. lmk what you think of the re-wording.

**Comment by verythorough** [comment]
I want to have a `whoa!` emoji reaction option for that Codecov comment. 😆

**Comment by erquhart** [comment]
Yeah Jessica if you could dramatically increase our coverage as a part of this docs PR that'd be great

**Pull request merged**